escapades_frontend/src/components/HomeImg/CoverImg.tsx

```diff
@@ -0,0 +1,19 @@
+import { FC } from "react";
+import classes from "./CoverCard.module.scss";
+
+type CoverCardProps = {
+  url: string;
+  text: string;
+};
```

Comment:
As discussed, I would prefer to use interface here. However it works as well.

Reply:
as you want, they are very similar anyway
here's a good article on this topic https://www.totaltypescript.com/type-vs-interface-which-should-you-use